components/article/client/view.coffee

```diff
@@ -78,46 +78,13 @@ module.exports = class ArticleView extends Backbone.View
     @checkEditable()
 
   maybeFinishedLoading: ->
-    if @loadedArtworks and not @loadedImageHeights
-      @setupMaxImageHeights()
-    else if @loadedArtworks and @loadedImageHeights
+    if @loadedArtworks and @loadedCallouts and @loadedImageHeights
```

Comment:
I think at this point we don't have `@loadedCallouts`, though https://github.com/artsy/force/pull/782 should change that